modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -0,0 +1,170 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="installing-the-sriov-fec-operator_{context}"]
+= Installing the OpenNESS SR-IOV Operator for Wireless FEC Accelerators
+
+The role of the OpenNESS SR-IOV Operator for Wireless FEC Accelerators is to orchestrate and manage the devices exposed by a range of Intel vRAN FEC acceleration hardware within the {product-title} cluster.
+
+One of the most compute-intensive 4G/LTE and 5G workloads is RAN layer 1 (L1) forward error correction (FEC). FEC resolves data transmission errors over unreliable or noisy communication channels. FEC technology detects and corrects a limited number of errors in 4G/LTE or 5G data without the need for retransmission.
+
+The FEC devices are provided by the Intel® FPGA PAC N3000 FPGA and the Intel® vRAN Dedicated Accelerator ACC100 for the vRAN use case.
+
+[NOTE]
+====
+The Intel® FPGA PAC N3000 FPGA requires flashing with an 4G/LTE or 5G bitstream.
+====
+
+The OpenNESS SR-IOV Operator for Wireless FEC Accelerators provides functionality to create desired virtual functions (VFs) for the FEC device, binds them to appropriate drivers, and configures the VFs queues for desired functionality in 4G/LTE or 5G deployment.
+
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the {product-title} CLI or the web console.
+
+[id="installing-sriov-fec-operator-using-the-cli_{context}"]
+== Installing the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI
+
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by completing the following actions:
+
+.. Define the `vran-acceleration-operators` namespace by creating a file named `sriov-namespace.yaml` as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: vran-acceleration-operators
+    labels:
+       openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-namespace.yaml
+----
+
+. Install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators in the namespace you created in the previous step by creating the following objects:
+
+.. Create the following `OperatorGroup` CR and save the YAML in the `sriov-operatorgroup.yaml` file:
```

Comment:
For parallelism, why isn't this instruction like the previous one?

```
Define the `vran-operators` operator group by creating a file named `sriov-operatorgroup.yaml` as shown in the following example:
```

Reply:
I am following the same structure I observed in the existing documentation regarding how to create the SR-IOV operator.

Reply:
Thanks for letting me know the origin.  I'll put that on my list to fix.